environs/manual/winrmprovisioner/winrmprovisioner.go

```diff
@@ -64,7 +64,13 @@ function Get-Arch {
 }
 
 function Get-Ram {
-   return (gcim win32_physicalmemory).Capacity
+   # The comma is not a typo. It forces $capacity to be an array
```

Comment:
Perhaps it would be worthwhile adding why we want it as an array? In what situations would it return more than one value?

Reply:
Good point. I will amend the comment, but for posterity will add the details here as well.

The command:

```powershell
gcim win32_physicalmemory
```

may return multiple values if you have multiple memory slots. The sum of the ```Capacity``` property of those values is our total physical memory. If we only have one slot, the above command will return an object of the type ```win32_physicalmemory```, instead of an array. This change simply makes a common case for both single slot machines and multi slot machines. Saves us the trouble of checking the type of the return value from ```gcim```.

Powershell is fun that way.

Reply:
Comment amended.